update tutorial.md add tips section (#1061)
spotipy-dev/spotipy: TUTORIAL.md

E. You may see a window open in your browser asking you to authorize the application. Do so - you will only have to do this once.

F. Return to your terminal - you should see all of Taylor's albums printed out there.

## Accessing specific data

> The data returned is an object of type dictionary. Accessing the data is quite easy and then can be transformed into json and string format. Below is an example of what is returned from querying the API for a user's top tracks.

Reviewer: > can be transformed into json and string format.

The wording could be improved. It reads like the user has to transform the json output themselves.

```suggestion
The data returned is an object of type dictionary. Accessing the data is quite easy as the library already parses the JSON response. Below is an example of what is returned from querying the API for a user's top tracks.
```

> ```

Reviewer: Add language to code block

```suggestion
```python
```

                                              'NZ',
                                              'OM',
                                              'PA',
                                              'PE',
                                              'PH',
                                              'PL',
                                              'PS',
                                              'PT',
                                              'PY',
                                              'QA',
                                              'RO',
                                              'SA',
                                              'SE',
                                              'SG',
                                              'SK',
                                              'SV',
                                              'TH',
                                              'TN',
                                              'TR',
                                              'TW',
                                              'US',
                                              'UY',
                                              'VN',

>                                               'ZA'],

Reviewer: Please minimize this.

                     'disc_number': 1,
                     'duration_ms': 297146,
                     'explicit': False,
                     'external_ids': {'isrc': 'GBCEL1900123'},
                     'external_urls': {   'spotify': 'https://open.spotify.com/track/5r6fLJUE1yMtEZTe71oDyy'},
                     'href': 'https://api.spotify.com/v1/tracks/5r6fLJUE1yMtEZTe71oDyy',
                     'id': '5r6fLJUE1yMtEZTe71oDyy',
                     'is_local': False,
                     'name': 'Friday',
                     'popularity': 28,
                     'preview_url': 'https://p.scdn.co/mp3-preview/ea4517f0fb93fc6b15669b5acf52565e8db8bc65?cid=c6d8222701234542ab1296cb5a242c46',
                     'track_number': 1,
                     'type': 'track',
                     'uri': 'spotify:track:5r6fLJUE1yMtEZTe71oDyy'}],
    'limit': 20,
    'next': 'https://api.spotify.com/v1/me/top/tracks?limit=20&offset=20&time_range=short_term',
    'offset': 0,
    'previous': None,
    'total': 50}
```
To provide a clear way of access and reformatting we can write some code to grab some important information that we see fit for our use case:

> ```

Reviewer: ```suggestion
```python
```

def get_top_tracks():
    results = sp.current_user_top_tracks(time_range="short_term", limit=20)
    top_track_array = []
    top_tracks_ids = {}
    for i, item in enumerate(results["items"]):
        top_tracks_ids = {
            "tid": item["artists"][0]["id"],
            "song_name": item["name"],
            "artist_name": item["artists"][0]["name"],
            "url": item["artists"][0]["external_urls"],
            "uri": item["uri"],
        }
        top_track_array.append(top_tracks_ids)
    return results
```
This transforms the raw response into something useful for our/your program:

> ```

Reviewer: ```suggestion
```python
```
